templates/_asset-card-image.html

```diff
@@ -35,26 +35,29 @@
           </p>
         {% endif %}
         <p class="u-no-margin--bottom">
-          File type: <strong>.{{ asset.file_type or asset.file_path.split(".")[-1].lower() }}</strong>
+          File type: <strong>.{{ asset.file_type or asset.file_path.split(".")[-1] .lower() }}</strong>
```

Comment:
```suggestion
          File type: <strong>.{{ asset.file_type or asset.file_path.split(".")[-1] .lower() }}</strong>
```
There is a space here, not sure if this will cause issues